wagtail/admin/static_src/wagtailadmin/scss/components/_listing.scss

```diff
@@ -377,7 +375,7 @@ ul.listing {
         .button {
             background-color: $color-white;
             color: $color-teal;
-            border-color: $color-teal;
+            border-color: rgba(0, 0, 0, 0.25);
```

Comment:
I'm finding this to be low enough contrast that I don't reliably read these as being buttons. I played with it a tiny bit in the browser and these options seemed ok: `$color-teal-darker` `$color-teal-dark` `rgba(0, 0, 0, 0.4)`

Reply:
I'll have another look at it. Buttons are defined in multiple places and in various ways. The used `0.25` as it matches the 'more' button outline.

Reply:
Oh wow, now that I look at it you're right that the buttons are entirely inconsistent. I'd love to make a PR to fix that. I do still think that darkening the button borders is important, and I think it would make more sense for that to be grouped with color changes than button style consistency. But I'm ok to do it there if you'd rather not do it here.

For reference, now that I have this locally, here's `rgba(0, 0, 0, 0.35)` which I think is a pretty minimal change but makes me feel better:
<img width="387" alt="image" src="https://user-images.githubusercontent.com/1051450/46267696-f0d8c500-c525-11e8-9153-038ed3c597cd.png">
vs the current `rgba(0, 0, 0, 0.25)`:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/1051450/46268307-4d89af00-c529-11e8-941f-9cf3c2e385d5.png">
and the current on master:
<img width="385" alt="image" src="https://user-images.githubusercontent.com/1051450/46268813-5af46880-c52c-11e8-9067-d6ebfb007b87.png">